tabbycat/checkins/templates/CheckInStatusContainer.vue

```diff
@@ -233,6 +254,28 @@ export default {
       }
       return this.entitiesByTime
     },
+    entitiesByBreaking: function () {
+      const isBreaking = (entityByPresence) => {
+        if (entityByPresence.type === 'Adjudicator') {
+          return entityByPresence.breaking
+        } else if (this.breakingTeamShortNames.includes(entityByPresence.name)) {
```

Comment:
Could using team IDs be shorter and avoid needing to escape the strings?

Reply:
Will do